docs/source/guide/tutorials/base/how_to_train/anomaly_detection.rst

```diff
@@ -1,4 +1,190 @@
-Anomaly Classification model
+Anomaly Detection model
 ================================
 
-This tutorial demonstrates how to train, evaluate, and deploy a classification, detection, or segmentation model for anomaly detection in industrial or medical applications.
\ No newline at end of file
+This tutorial demonstrates how to train, evaluate, and deploy a classification, detection, or segmentation model for anomaly detection in industrial or medical applications.
+
+The process has been tested with the following configuration:
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-10980XE
+- CUDA Toolkit 11.1
+
```

Comment:
Please, add here links as note as below:
![image](https://user-images.githubusercontent.com/68103869/220469307-038ea7cf-58fc-4351-bcd2-1ad5735d7d0f.png)

Reply:
I haven't added link training process with additional parameters as I didn't really test it so not sure if it will work with anomaly tasks.